src/main/java/mate/hwdao/dao/impl/jdbc/ManufactureDaoJdbcImpl.java

```diff
@@ -0,0 +1,112 @@
+package mate.hwdao.dao.impl.jdbc;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.hwdao.dao.ManufacturerDao;
+import mate.hwdao.lib.Dao;
+import mate.hwdao.model.Manufacturer;
+import mate.hwdao.util.ConnectionUtil;
+
+@Dao
+public class ManufactureDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacture (name, country, isexist) VALUES (?, ?, ?)";
+        try (PreparedStatement preparedStatement
+                    = ConnectionUtil.getConnection()
+                .prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            preparedStatement.setBoolean(3, true);
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            while (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException ex) {
+            throw new RuntimeException("Saving manufacturer " + manufacturer + " failed", ex);
```

Comment:
you've put checkmark that you checked solution with checklist, but there is still a mistake 
![image](https://user-images.githubusercontent.com/58385506/104578581-91a20e80-5663-11eb-8602-0c96a1eeee1e.png)
change the exception type

Reply:
done